src/EditorFeatures/CSharpTest/SplitComment/SplitCommentCommandHandlerTests.cs

```diff
@@ -0,0 +1,249 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Editor.UnitTests.SplitComment;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.SplitComment
+{
+    [UseExportProvider]
+    public class SplitCommentCommandHandlerTests : AbstractSplitCommentCommandHandlerTests
```

Comment:
💡 Need tests for `////` (this is a fairly common single-line prefix, which should probably be preserved on splitting the same way whitespace is)

Reply:
Done!